src/types/onyx/BankAccount.ts

```diff
@@ -49,7 +51,7 @@ type AccountData = {
 
 type BankAccount = {
     /** The bank account type */
-    accountType?: string;
+    accountType?: typeof CONST.PAYMENT_METHODS.BANK_ACCOUNT;
```

Comment:
Is this really correct?

Reply:
@Skalakid Please check it once you are back

Reply:
I checked the codebase and i can't find any other string that `accountType` can be. Also I tested the app and all bank accounts are type of `CONST.PAYMENT_METHODS.BANK_ACCOUNT`. The change was suggested by @blazejkustra [here](https://github.com/Expensify/App/pull/27923#discussion_r1333037769). Błażej do you have any other argument why we shoudn't use `string` type here?

Reply:
No, it's just to be precise. Not much benefits out of it 🤷